scripts/docstrings_checker.py

```diff
@@ -203,3 +198,27 @@ class GoogleDocstring(_check_docs_utils.GoogleDocstring):
     """.format(
         type=re_multiple_type,
     ), flags=re.X | re.S | re.M)
+
+    def match_param_docs(self):
+        """Returns the set of parameter names which are properly documented.
+
+        Returns:
+            set(str). A set of parameter names which are properly defined
+            in docstring.
+        """
```

Comment:
Add a comment with the link to the pylint method from where this is modified from. I guess its [this](https://github.com/PyCQA/pylint/blob/e89c361668aeead9fd192d5289c186611ef779ca/pylint/extensions/_check_docs_utils.py#L585) (Not sure)